app/Models/Repository.php

```diff
@@ -15,7 +16,8 @@
  */
 class Repository extends Model
 {
-    use SoftDeletes;
+    use SoftDeletes,
+        HasPermissions;
```

Comment:
it is better to 
```
use SoftDeletes;
use HasPermissions;
```

Reply:
no

Reply:
we don't need to fix it now. only take care. it's better syntax. ❤️